src/compo/viirs_l1bnc2ioda.py

```diff
@@ -157,8 +153,7 @@ def _read(self):
             solar_aa = geo_ncd.groups['geolocation_data'].variables['solar_azimuth'][:].data.ravel()
             sensor_za = geo_ncd.groups['geolocation_data'].variables['sensor_zenith'][:].data.ravel()
             sensor_aa = geo_ncd.groups['geolocation_data'].variables['sensor_azimuth'][:].data.ravel()
-            sin_va = np.sin(sensor_za*deg2rad) * earth_radius / (earth_radius + orbit_height)
-            sensor_va = np.arcsin(sin_va) * rad2deg
+            sensor_va = compute_scan_angle(sensor_za, orbit_height, sensor_za)
```

Comment:
this function needs a cleanup btw,,, the first argument is not used... I will remove in a separate issue and PR and correct here, so it would look like `compute_scan_angle(orbit_height, sensor_za)`

Reply:
this is likely to occur after this PR is merged